templates/three/packages/client/src/App.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Canvas, Color, ThreeElements, useThree } from "@react-three/fiber";
+import { useComponentValue, useEntityQuery } from "@latticexyz/react";
+import { getComponentValueStrict, Has } from "@latticexyz/recs";
+import { useMUD } from "./MUDContext";
+import { useKeyboardMovement } from "./useKeyboardMovement";
+
+const Plane = (props: ThreeElements["mesh"]) => {
+  return (
+    <mesh {...props}>
+      {/* eslint-disable-next-line react/no-unknown-property */}
```

Comment:
These unknown types didn't come up in v2Sandbox, maybe this is missing a package